src/models/simple_metabolism_model.py

```diff
@@ -135,22 +130,21 @@ def get_iob_from_sbr(self, sbr_actual):
         num_hours_post_t0 = 8
         minutes_per_pump_pulse = 5
 
-        # E.g. 1 pulse every 5 minutes: pulses/hr = 60 min/hr / 5 min/pulse
+        # E.g. 1 pulse every 5 minutes: 12 pulses/hr = 60 min/hr / 5 min/pulse
         num_basal_pulses_per_hour = int(MINUTES_PER_HOUR / minutes_per_pump_pulse)
 
         # TODO: CS - this is unrealistic pump behavior since pumps deliver in increments of U/pulse
```

Comment:
this might be worth a conversation. I think this is fine for MVP given that we are trying to model the approximate amount of insulin that gets delivered every 5 minutes.

Reply:
Yes, I agree it's totally fine MVP. I'm 1) being a bit pedantic since in this case the steady state basal properties matter little even if this is modeled in pulses like a real pump (I coded this up just to check and it's just a bit chattery around a mean) and 2) wanting to simply have a note for future reference of the assumption.